contents/handbook/engineering/developing-locally.md

```diff
@@ -186,12 +204,19 @@ Assuming Node.js is installed, run `yarn --cwd plugin-server` to install all req
 1. Install requirements with pip
 
     If your workstation is ARM-based (e.g. Apple Silicon), the first time your run `pip install` you must pass it custom OpenSSL headers:
-
+    - On macOs
     ```bash
     brew install openssl brotli
     CFLAGS="-I /opt/homebrew/opt/openssl/include" LDFLAGS="-L /opt/homebrew/opt/openssl/lib" GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1 GRPC_PYTHON_BUILD_SYSTEM_ZLIB=1 pip install -r requirements.txt
     ```
 
+    - On Linux
+    ```bash
+    apt-get update
+    apt-get install brotli -y
```

Comment:
```suggestion
    sudo apt-get install brotli libpq-dev -y
```

I also needed to install `libpq-dev` in order to get the `psycopg2-binary` package to build.